### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Changelog

```markdown
### Deprecated
- Passing a null offset to `AuditReader::findRevisionHistory`
```

### CODE REVIEW

File: src/Model/AuditReader.php [hunk 1]
```diff
@@ -47,8 +47,21 @@ public function find(string $className, $id, $revisionId): ?object
         }
     }
 
+    /**
+     * NEXT_MAJOR: Change the default limit value to `null`.
```
Comment (VincentLanglet) [inline src/Model/AuditReader.php:51]: Consistent with https://github.com/sonata-project/SonataAdminBundle/pull/7959

Comment (jordisala1991) [reply src/Model/AuditReader.php:51]: Why allow null? should the change be only the deprecation of null?

Comment (VincentLanglet) [reply src/Model/AuditReader.php:51]: What if you want to get all the revision history ?

The auditReader allow to use null limit, so why not keeping the feature ?

Comment (jordisala1991) [reply src/Model/AuditReader.php:51]: OH I see, the deprecation is for offset, and the allow null is for limit. 👍

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**